pint/registry.py

```diff
@@ -325,6 +330,13 @@ def _define_single_adder(self, key, value, unit_dict, casei_unit_dict):
         if casei_unit_dict is not None:
             casei_unit_dict[key.lower()].add(key)
 
+    def _define_alias(self, definition, unit_dict, casei_unit_dict):
+        unit = unit_dict[definition.name]
+        unit.add_aliases(*definition.aliases)
+        for alias in unit.aliases:
+            unit_dict[alias] = unit
+            casei_unit_dict[alias.lower()].add(alias)
```

Comment:
I could not figure out how casei_unit_dict works - or find any tests for it. So I just copied what already is done for new units.

Reply:
That is fine